samza-core/src/main/java/org/apache/samza/runtime/RemoteApplicationRunner.java

```diff
@@ -35,12 +37,32 @@
  */
 public class RemoteApplicationRunner extends AbstractApplicationRunner {
 
-  private static final Logger log = LoggerFactory.getLogger(RemoteApplicationRunner.class);
+  private static final Logger LOG = LoggerFactory.getLogger(RemoteApplicationRunner.class);
```

Comment:
QQ: How is RemoteApplicationRunner invoked? Isn't it through some wrapper shell script and user never sees this class? Need clarification as we have been back-and-forth on this interface discussion.